mock-core-configs/etc/mock/templates/openeuler-22.03.tpl

```diff
@@ -0,0 +1,70 @@
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ openEuler-rpm-config openEuler-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['dist'] = 'oe2203'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '22.03'
+config_opts['package_manager'] = 'dnf'
+config_opts['description'] = 'openEuler 22.03'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+
+config_opts['dnf.conf'] = """
+[main]
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+metadata_expire=0
+mdpolicy=group:primary
+best=1
+install_weak_deps=0
+protected_packages=
+module_platform_id=platform:oe2203
+user_agent={{ user_agent }}
+
+[OS]
+name=OS
+baseurl=http://repo.openeuler.org/openEuler-22.03-LTS/OS/$basearch/
+enabled=1
+gpgcheck=1
+gpgkey=http://repo.openeuler.org/openEuler-22.03-LTS/OS/$basearch/RPM-GPG-KEY-openEuler
```

Comment:
This should be a GPG key file that's shipped in [`distribution-gpg-keys`](https://github.com/xsuchy/distribution-gpg-keys) rather than some web URL.

Please send a pull request to add the openEuler GPG key to the package and adjust this to use those keys like the others do.

Reply:
Wow! Thanks for your super fast review! I will address soon.

https://github.com/xsuchy/distribution-gpg-keys/pull/73